packages/langium/src/documents/document-builder.ts

```diff
@@ -63,3 +93,6 @@ export class DefaultDocumentBuilder implements DocumentBuilder {
     }
 
 }
+function isAffected(document: LangiumDocument, changedUri: string): boolean {
+    return changedUri !== document.textDocument.uri;
+}
```

Comment:
Is this a placeholder? Change affection should be overridable (a template method or separate service) and should consider references between documents.

Reply:
Moved to service implementation. 
> and should consider references between documents.

Yes, I know. I would like to keep it that simple and create tests first. Afterwards I will provide a more complex and performant version.